Cargo.toml

```diff
@@ -15,4 +15,4 @@ categories = ["algorithms", "no-std"]
 edition = "2018"
 
 [dependencies]
-crc-catalog = "1.1.1"
+crc-catalog = "2.0.0"
```

Comment:
Looks like this bumped our MSRV. I'm gonna go ahead and revert this crate to 2018 edition.

Reply:
Published `crc-catalog` 2.0.1 with an MSRV of 1.46.